eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -96,12 +96,14 @@
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
   <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientInstantiationCheck" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
   <suppress checks="com.azure.tools.checkstyle.checks.ThrownClientLoggerCheck" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.HttpPipelinePolicyCheck" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
```

Comment:
We have a lot of the lines for suppressions of files that are for our Track 1 APIs (com.microsoft.*).  I think a lot of these lines could be removed and replaced with a single line. (I read that "checks" can take a regex,)
```xml
<suppress checks="com\.azure\.tools\.checkstyle\.checks\.*" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
```
Thoughts,  @JonathanGiles? Or do you prefer it be explicit for all these custom checkstyle rules.

Supression Files Docs: https://checkstyle.sourceforge.io/config_filters.html#SuppressionFilter_Notes

Reply:
I would love to see this change. Please make it so @mssfang.